kinder/pkg/build/bits/initBits.go

```diff
@@ -280,7 +280,7 @@ func configureKubelet(c *BuildContext) error {
 
 	// ensure we don't fail if swap is enabled on the host
 	if err := c.RunInContainer("/bin/sh", "-c",
-		`echo "KUBELET_EXTRA_ARGS=--fail-swap-on=false" >> /etc/default/kubelet`,
+		`echo "FailSwapOn: false" >> /var/lib/kubelet/config.yaml`,
```

Comment:
kubeadm init and join regenerate this file and it will be overwritten.

the /etc/default/kubelet override does works but it's for flags only.

i think the value should be added here instead:
https://github.com/kubernetes/kubeadm/blob/05f9b96cf83d73ed2177815487eba90517b66708/kinder/pkg/kubeadm/config.go#L200-L216